mifosng-provider/src/main/java/org/mifosplatform/commands/service/CommandWrapperBuilder.java

```diff
@@ -2353,4 +2353,29 @@ public CommandWrapperBuilder deleteMap(final Long mapId) {
         this.href = "/entitytoentitymapping/" + mapId;
         return this;
     }
+
+    public CommandWrapperBuilder createPaymentType() {
+        this.actionName = "CREATE";
+        this.entityName = "PAYMENTTYPE";
+        this.entityId = null;
+        this.href = "/paymenttype";
```

Comment:
move all reference to paymenttype to a constant (becomes easier for you to change the resource name)